src/FormulateForm.vue

```diff
@@ -32,7 +32,7 @@ export default {
   name: 'FormulateForm',
   model: {
     prop: 'formulateValue',
-    event: 'input'
+    event: 'update:modelValue'
```

Comment:
https://v3.vuejs.org/guide/migration/v-model.html#overview

Reply:
I suppose we'll need to manually emit `input` events now too for backwards compatibility